src/lucky/route_not_found_handler.cr

```diff
@@ -1,9 +1,17 @@
+# This HTTP::Handler takes in the current `context`,
+# then checks to see if a `fallback_action` has been defined to render that action first.
+# If no fallback has been defined, then raise a `Lucky::RouteNotFoundError` exception.
+#
+# This handler should be used after the `Lucky::RouteHandler`.
+#
+# See `Lucky::Routeable.fallback` for implementing the `fallback_action`.
 class Lucky::RouteNotFoundHandler
   include HTTP::Handler
   class_property fallback_action : Lucky::Action.class | Nil
 
   def call(context)
     if has_fallback?(context)
+      Lucky.logger.debug("Handled by fallback in #{fallback_action.name.to_s.colorize.bold}")
```

Comment:
This will automatically be bolded, etc when passed in as data. That way it can be formatted as JSON, pretty formatted, etc.

```
Lucky.logger.debug(handled_by_fallback: fallback_action.name.to_s)
```

Reply:
I actually modeled this after [this line](https://github.com/luckyframework/lucky/blob/master/src/lucky/route_handler.cr#L9) which does the same. Should that one be changed too then? (separate PR)

Reply:
Ah yes it should also change!